w1_d2_2_bootstrap_css/README.md

```diff
@@ -0,0 +1,100 @@
+# HTML/CSS Review, UX & Bootstrap
+| Objectives |
+| :--- |
+| Explain why UX is important as a developer |
+| Build a static webpage with HTML and CSS (review) |
+| Add the Bootstrap library to your projects |
+
+## UX & UI
+  * UX = User Experience
+  * UI = User Interface
+
+#### Key Takeaways
+  * UX encompasses all aspects of the end user's interaction, including product design, usability, and interface layout
+  * UI refers to the visual elements (typography, buttons, forms, etc.)
+
+#### Why is UX important?
+
+  As developers, we want to make web applications that are *useable* and make people *happy*.
+
+## Bootstrap
+  * [Bootstrap Docs](http://getbootstrap.com/css)
+  * CSS library developed by Twitter
+  * Best known for responsive grid system
+
+#### Why Use Bootstrap
+  * Powerful tool for quickly implementing UX and responsive design
+  * Enforces [semantic CSS class names](https://css-tricks.com/semantic-class-names)
+  * Can be [minimal](http://dartjobs.herokuapp.com) or [customized](https://www.stitchfix.com)
+
+#### How to Use Bootstrap
+  * Add the the viewport meta tag and the [Bootstrap CDN](http://getbootstrap.com/getting-started/#download) to the `<head></head>` of your HTML file
+
+  ```
+  <head>
+      <meta name="viewport" content="width=device-width, initial-scale=1">
+      <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap.min.css">
+  </head>
+  ```
+
+  * You can also use this [Bootstrap boilerplate](https://github.com/sf-wdi-19-20/staging-modules/tree/master/w1_d2_2_bootstrap_css/bootstrap_boilerplate) to get started
```

Comment:
Good call -- I'll update that now! Just so everyone knows, the "non-staging" links won't work until this is merged with master, but I'll double check them once it's merged.